AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/ConfigurationTests/AWSCognitoAuthPluginConfigTests.swift

```diff
@@ -188,4 +188,61 @@ class AWSCognitoAuthPluginConfigTests: XCTestCase {
         }
     }
 
+    /// Test Auth configuration order of execution of apis
+    ///
+    /// - Given: Given valid config for user pool and identity pool
+    /// - When:
+    ///    - I call different auth apis after configuration
+    /// - Then:
+    ///    - I should get the result back based on the order in which the calls where made.
+    ///
+    func testAPIExecutionOrder() throws {
+        let plugin = AWSCognitoAuthPlugin()
```

Comment:
Have you verified that this test fails without your operationQueue changes? Since you're invoking everything serially, it's not clear that the previous concurrency behavior would result in a test failure.

Reply:
Yes, the test without the change will fail.